designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/IconButton.kt

```diff
@@ -122,6 +122,28 @@ fun IconButton(
     }
 }
 
+@Composable
+fun BottomSheetIconButton(
```

Comment:
Are we reusing this button?

Reply:
this button will be reused for all bottomSheet variations but I could probably make it internal